### ISSUE TITLE
IOWait problem in kubernetes after the last update

### ISSUE THREAD

**Issue by 5hin0bi** [openappsec/openappsec#103]
After the image was updated (because of using the latest tag), I have updated the whole helm chart so that it started postgres etc.

After the update I've noticed that at the 12 AM server time something started to cause the IOWait problems:
![image](https://github.com/openappsec/openappsec/assets/94184844/bfc8c150-5b03-4488-97d9-acff8a1a6ff5)

Here is the iotop ouptut:
![image](https://github.com/openappsec/openappsec/assets/94184844/169451ec-2b2a-4c91-9142-da1877f50941)

Is this something expected? Does it require faster disk to fade? It keeps totally fine for 2 days, on the second day at 12 AM it starts something.

Is there any temporary workaround? It causes slower responses to our clients.

P.S.: it happened twice, I replaced the k8s node and started new appsec pod, it was OK for 2 days then such situation happened again.
Yet, I've replaced the node again and awaiting the same behavior in 2 days. So it would be the 3rd case of such IOWait problem.

**Comment by 5hin0bi** [comment]
FYI: it happened twice, I replaced the k8s node and started new appsec pod, it was OK for 2 days then such situation happened again.
Yet, I've replaced the node again and awaiting the same behavior in 2 days.

**Comment by 5hin0bi** [comment]
Hi @orianelou ,

Thank you for your response.
I think we may close the issue since I haven't faced the problem AFTER I've upgraded the version to 1.1.5.

The order:
1. We were using 1.1.3 but then suddenly I've noticed some problems since the image was repulled to latest but the chart itself was not updated.
2. I've updated it to 1.1.4 and the problem I described started.
3. I've noticed 1.1.5 was released slightly before I've opened the issue, so I've upgraded the chart to v1.1.5 and the issue seems to be gone.

Not very sure what rly caused it. I don't think wasting your time on this is good. Thank you for the reply anyways!
